pwem/protocols/protocol_align_movies.py

```diff
@@ -105,6 +105,16 @@ def _defineAlignmentParams(self, form):
                       label="Save aligned micrograph",
                       expertLevel=pwcts.LEVEL_ADVANCED)
 
+        form.addHidden('tomographySplitFrames', params.BooleanParam,
+                       default=getattr(self, 'tomographySplitFrames', False))
+        form.addParam('splitEvenOdd', params.BooleanParam,
+                      default=False,
+                      label='Split & sum odd/even frames?',
+                      condition='tomographySplitFrames',
+                      expertLevel=pwcts.LEVEL_ADVANCED,
+                      help='(Used for cryoCARE denoising data preparation). If set to Yes, 2 additional tilt series '
```

Comment:
Even more, if it is only for cryoCARE, it should be analyzed where it is the best place for this option.

Reply:
Theoretically, any denoiser or maybe any other method might use this even / odd data. So I'd say is not cryoCARE specific but needed by cryoCARE.

Reply:
I guess this will depend on the specific algorithm used by the denoiser, I don't know about that, but probably not all of them will require the same input.

Reply:
I'm not an expert either, I just see that splitting even and odd has been done in the past by xmipp team for something else and is the same kind of splitting. But time will tell.